### DISCUSSION TITLE
in this 6 version much much issues.

### DISCUSSION THREAD

**Discussion by sameerhabib72** [Power-Components/livewire-powergrid#1740] [Q&A]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### PHP Version

8.2.12

### PowerGrid

6

### Laravel

11.28.1

### Livewire

3.5.12

### Alpine JS

3.5.12

### Theme

Bootstrap

### Describe the bug.

in this 6 version much much issues not showing action buttons showing by default unknow message can same one help to install version of 5 because version 5 is working good  

### To Reproduce...

First click on "FOO" then....

### Extra information

```markdown
<?php
 //...
```

**Comment by luanfreitasdev** [comment]
Please provide more details about what problems you are having.

**Reply by sameerhabib72** [reply]
top on table loading  error undefined Loading..., action column action  button error undefined  and table header color 
![Screenshot (103)](https://github.com/user-attachments/assets/873fb840-8807-4a55-b14e-a7b738809ddc)

**Reply by backstageel** [reply]
This problem is because you forgot to update your Javascript and CSS files when you upgraded to V6.0

**Reply by piraticame11** [reply]
Finish the configurations, I solved this problem after I did this.
![image](https://github.com/user-attachments/assets/a9e15ddf-e8b7-46dc-a264-6211f303a09d)

**Comment by ashek1412** [comment]
same problem when using inside filament admin.

![image](https://github.com/user-attachments/assets/8759ab88-f7f8-4476-9b20-339378c23a53)
